pkg/testcase/cluster.go

```diff
@@ -114,3 +114,30 @@ func checkAndPrintAgentNodeIPs(agentNum int, agentIPs []string, isWindows bool)
 		Expect(agentIPs).Should(BeEmpty())
 	}
 }
+
+func ClusterReset() {
+	product, err := shared.Product()
+	Expect(err).NotTo(HaveOccurred())
+
+	g := GinkgoT()
+	cluster := factory.ClusterConfig(g)
+	ip := cluster.ServerIPs[0]
+
+	switch {
+	case product == "k3s":
+		cmd := fmt.Sprintf("sudo %s server --cluster-reset", product)
+		res, err := shared.RunCommandOnNode(cmd, ip)
+		Expect(err).NotTo(HaveOccurred())
+		Expect(res).To(ContainSubstring("Managed etcd cluster"))
+		Expect(res).To(ContainSubstring("has been reset"))
+	case product == "rke2":
+		cmd := fmt.Sprintf("sudo %s server --cluster-reset", product)
+		_, err := shared.RunCommandOnNode(cmd, ip)
+		Expect(err).To(HaveOccurred())
+		Expect(err.Error()).To(ContainSubstring("Managed etcd cluster"))
+		Expect(err.Error()).To(ContainSubstring("has been reset"))
+	default:
+		shared.ReturnLogError("unsupported product: %s\n", product)
```

Comment:
you cant use this here, lint is also complaining not just me lol

Reply:
seriously now, your func does not return error, so if you have a default as error its not the best strategy use a function that returns error here

Reply:
i added a workaroud for you on the snippet above